### PR TITLE
Fix inability to deploy with helm due to missing certificates

### PR DESCRIPTION
**Pull request by connorgorman**
## Description

We need to generate the central db and central db certs in roxctl which is used in the k8s based install

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

oldest openshift and ui tests

### CODE REVIEW

File: deploy/common/k8sbased.sh [hunk 1]
```diff
@@ -253,6 +253,8 @@ function launch_central {
 
     echo "Deploying Central..."
 
+    $(git rev-parse --show-toplevel)/scripts/ci/store-artifacts.sh store_artifacts "${k8s_dir}/central-deploy"
```
Comment (gavin-stackrox) [inline deploy/common/k8sbased.sh:256]: nit:
```suggestion
    "$(git rev-parse --show-toplevel)/scripts/ci/store-artifacts.sh" store_artifacts "${k8s_dir}/central-deploy"
```

**Review by gavin-stackrox — APPROVED** [review]

File: pkg/renderer/helm_values.go [hunk 2]
```diff
@@ -403,9 +403,6 @@ var (
 	publicValuesTemplate = template.Must(
 		helmTemplate.InitTemplate("values-public.yaml").Parse(publicValuesTemplateStr))
 
-	privateValuesTemplate = template.Must(
```
Comment (dashrews78) [inline pkg/renderer/helm_values.go:406]: should probably remove `privateValuesYamlPostgresTemplateStr` while you are at it.

**Review by dashrews78 — APPROVED** [review]

**Review by c-du — APPROVED** [review]

**Pull request merged**